uint8 axis
---
bool success